target/mips/cpu.c

```diff
@@ -207,6 +207,20 @@ static void mips_cpu_synchronize_from_tb(CPUState *cs, TranslationBlock *tb)
     mips_update_pc(env, tb->pc, /*can_be_unrepresentable=*/false);
     env->hflags &= ~MIPS_HFLAG_BMASK;
     env->hflags |= tb->flags & MIPS_HFLAG_BMASK;
+
+#ifdef TARGET_CHERI
```

Comment:
We could probably remove the TARGET_CHERI? I think clearing it for the current CPU is also fine if you have mttcg.